docs/understanding-radicle/faq.md

```diff
@@ -72,7 +71,7 @@ GitHub replica.
 ## Where is my data stored?
 On the Radicle network, content is distributed peer-to-peer via a process called
 gossip. This means that peers self-host their own content — and the content of
-any peers they are interested in — locally on their machine. It also means that
+any peers they are interested in — locally on their machine in a [Git monorepo][hw]. It also means that
 whenever your data is published to the network, it can be replicated and stored
 on another peer's machine.
```

Comment:
```suggestion
any peers they are interested in — locally on their machine in a [Git monorepo][hw].
It also means that whenever your data is published to the network, it can be
replicated and stored on another peer's machine.
```